Gruntfile.js

```diff
@@ -26,36 +26,21 @@
 module.exports = function (grunt) {
     "use strict";
     
-    /**
-     * Get options for r.js parametrized by locale.
-     *
-     * @param {string} locale
-     * @return {object}
-     */
-    var getRequireOptions = function (locale) {
-        return {
-            options: {
-                baseUrl: "src/",
-                mainConfigFile: "src/js/config.js",
-                name: "js/init-build",
-                out: "build/js/init-build-" + locale + ".js",
-                // optimize: "none",
-                paths: {
-                    "react": "../bower_components/react/react-with-addons.min",
-                    "JSXTransformer": "../bower_components/jsx-requirejs-plugin/js/JSXTransformer"
-                },
-                stubModules: ["jsx"],
-                exclude: ["JSXTransformer"],
-                useStrict: true,
-                config: {
-                    i18n: {
-                        locale: locale
-                    }
-                }
-            }
-        };
-    };
+    if (grunt.cli.tasks[0] === "dev") {
```

Comment:
There is a bug here where if you type anything but `grunt dev`, webpack tries to build all locales. I'm looking into it. For now, avoid using anything but `grunt compile` and `grunt dev`